src/lib/structures/expr.ml

```diff
@@ -1990,15 +1990,15 @@ module Triggers = struct
 
     | { f = Op (Access a1) ; xs=[t1]; _ },
       { f = Op (Access a2) ; xs=[t2]; _ } ->
-      let c = Stdlib.compare a1 a2 in (* should be Hstring.compare *)
+      let c = Hstring.compare a1 a2 in (* should be Hstring.compare *)
```

Comment:
```suggestion
      let c = Hstring.compare a1 a2 in
```

Reply:
I removed it another PR ;)

Reply:
I may be missing something but given that this is the PR that fixes the issue identified by the comment, it should also be the one removing the comment.

Reply:
I know but in fact this PR is extracted from #1098 . I can do it twice if you want...

Reply:
It would have been better to avoid the inconsistent state one way or another. That said, this being replaced by `Uid.compare` (I suppose?) in that PR makes sense; I have to review PRs as they are and can't really infer what "another PR" refers to — thanks for the clarification.